spec/app/ics-032-interchain-queries/README.md

```diff
@@ -0,0 +1,279 @@
+---
+ics: 32
+title: Interchain Queries
+stage: Draft
+category: IBC/APP
+requires: 25, 26
+kind: instantiation
+author: Ali Zahid Raja <ali@polymerlabs.org>, Ehsan Saradar <ehsan@quasar.fi>
+created: 2022-06-22
+modified: 2022-08-30
+---
+
+## Synopsis
+
+This standard document specifies chain structure, packet flow and authentication logic for the Interchain Queries module. This allows for one blockchain to query for on-chain data from another block chain via ABCI queries.
+
+### Motivation
+
+Interchain Queries enable blockchains to query the state of an account on another chain without the need for ICA auth. ICS-27 Interchain Accounts is used for IBC transactions, e.g. to transfer coins from one interchain account to another, whereas Interchain Queries is used for IBC Query, e.g. to query the balance of an account on another chain. 
+In short, ICA is cross chain writes while ICQ is cross chain reads.
+
+### Definitions 
+
+- `Host Chain`: The chain where the query is sent. The host chain listens for IBC packets from a controller chain that contain instructions (e.g. cosmos SDK messages) that the interchain account will execute.
+- `Querier Chain`: The chain sending the query to the host chain. The controller chain sends IBC packets to the host chain to query information.
+- `Interchain Query`: An IBC packet that contains information about the query in the form of ABCI RequestQuery
+
+The chain which sends the query becomes the controller chain, and the chain which receives the query and responds becomes the host chain for the scenario.
```

Comment:
"controller chain" should be "querying chain"

Reply:
Resolved here: https://github.com/strangelove-ventures/async-icq/pull/8
Moving this readme out of `cosmos/ibc` and into `strangelove-ventures/async-icq`